modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -7,44 +7,52 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Blacklist
->s|Summary      | Blocks a list or range of IP addresses
+>s|Summary      | Blocks access from a range of IP addresses to an API endpoint
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available  | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
-The IP blacklist policy can control whether a list or range of IP addresses can access and interact with a configured API endpoint. You use one of the following options to blacklist addresses for your API:
+The IP Blacklist policy controls access to a configured API endpoint from a range of IP addresses. You can blacklist access based on the IP address of the request
+or based on the origin IP address of the `x-Forwarded-For` header.
 
-* Option 1: Blacklist access based on the IP address of the request
-* Option 2: Blacklist access based on the origin IP address of the x-Forwarded-For header
+//Nauel--will users know what this header is?
 
-== Option 1: Blacklist access based on the IP address of the request
+== How This Policy Works
 
-. Click the instance number of an API.
-+
-. On the API dashboard, click Policies.
-+
-The list of any applied policies appears.
+// Nahuel--anything to add here?
+
+[[specifying_ip_address]]
+== Specifying IP Addresses to Blacklist in the Policy
+
+You can specify the IP addresses to block in the policy using the following ways:
+
+* Blacklist access based on the IP address of the request
+
+* Blacklist access based on the origin IP address of the x-Forwarded-For header
```

Comment:
Also blacklist access based on the origin IP address that is determined by resolving a specific expression (`x-forwarded-for` is a particular case of this).

Reply:
Done! Thanks!